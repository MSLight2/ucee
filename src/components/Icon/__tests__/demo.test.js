import { demoTest } from '../../../tests/shared/demoTest';
import IconDemo from '../demo';

demoTest('Icon', IconDemo);