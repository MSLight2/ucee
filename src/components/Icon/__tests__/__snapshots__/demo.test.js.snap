// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon demo test 1`] = `
Array [
  <i
    className="ucee-icon ucee-icon-taxi"
    style={Object {}}
  />,
  <i
    className="ucee-icon ucee-icon-deliver"
    style={Object {}}
  />,
  <i
    className="ucee-icon ucee-icon-deliver"
    style={
      Object {
        "fontSize": "26px",
      }
    }
  />,
  <i
    className="ucee-icon ucee-icon-deliver"
    style={
      Object {
        "fontSize": "2vw",
      }
    }
  />,
  <i
    className="ucee-icon ucee-icon-deliver"
    style={
      Object {
        "color": "#f90",
        "fontSize": "30px",
      }
    }
  />,
  <i
    className="my-icon my-icon-deliver icon-u"
    style={Object {}}
  />,
  <div
    className="ucee-icon__box"
  >
    <span
      className="ucee-icon--dot"
    />
    <i
      className="ucee-icon ucee-icon-notice"
      style={
        Object {
          "fontSize": "30px",
        }
      }
    />
  </div>,
  <div
    className="ucee-icon__box"
  >
    <span
      className="ucee-icon--badge"
    >
      99+
    </span>
    <i
      className="ucee-icon ucee-icon-notice"
      style={
        Object {
          "fontSize": "30px",
        }
      }
    />
  </div>,
  <div
    className="ucee-icon__box"
  >
    <span
      className="ucee-icon--badge"
    >
      25
    </span>
    <i
      className="ucee-icon ucee-icon-notice"
      onClick={[Function]}
      style={
        Object {
          "fontSize": "30px",
        }
      }
    />
  </div>,
  <i
    className="ucee-icon ucee-icon-logo.png"
    style={Object {}}
  />,
  <div
    className="ucee-icon"
  >
    <img
      alt=""
      className="ucee-icon__img"
      src="https://img.yzcdn.cn/vant/user-active.png"
    />
  </div>,
]
`;
